tests/vmi_configuration_test.go

```diff
@@ -659,14 +687,10 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 					Should(ContainSubstring("EFI OVMF rom missing"))
 			default:
 				libwait.WaitUntilVMIReady(vmi, loginTo)
-				By(msg)
-				domXml, err := libdomain.GetRunningVirtualMachineInstanceDomainXML(virtClient, vmi)
-				Expect(err).ToNot(HaveOccurred())
-				Expect(domXml).To(MatchRegexp(fileName))
 			}
 		},
-			Entry("[test_id:1668]should use EFI without secure boot", Serial, alpineWithUefiWithoutSecureBoot, console.LoginToAlpine, "Checking if UEFI is enabled", `OVMF_CODE(\.secboot)?\.fd`),
-			Entry("[test_id:4437]should enable EFI secure boot", Serial, fedoraWithUefiSecuredBoot, console.SecureBootExpecter, "Checking if SecureBoot is enabled in the libvirt XML", `OVMF_CODE\.secboot\.fd`),
+			Entry("[test_id:1668]should use EFI without secure boot", Serial, alpineWithUefiWithoutSecureBoot, console.LoginToAlpine),
```

Comment:
These changes reduce coverage, can you check if secure boot is enabled at runtime instead? A fedora with secure boot might still be able to boot without it.